### DISCUSSION TITLE
"NotImplementedError: pool is not implemented"  when loading  'detector.net' with netx.hdf5.Network for loihi inference

### DISCUSSION THREAD

**Discussion by gwgknudayanga** [lava-nc/lava-dl#346] [Q&A]
Hi Experts,

My detector network that i trained in slayer has some pooling layers. I exported the trained network to  'detector.net' format and then When tried to import that 'detector.net' for lava inference with netx.hdf5.Network it gives the error "NotImplementedError: pool is not implemented" . So does this mean we cannot run pooling layers in loihi2? 
If pooling layers are possible to deploy in loihi how should i proceed?

Thank you.

Thanks and Rgds,
Udayanga

**Comment by waseemshariff126** [comment]


**Reply by gwgknudayanga** [reply]
Hi ,
Thanks for your reply. 
In the slayer there are layers like " slayer.block.cuba.Pool". When I tried export this from slayer to lava with "export_hdf5" , it gives the error ""NotImplementedError: pool is not implemented". It seems that though they are available in slayer not available in lava.
